### PR TITLE
Generic API: start to handle polymorphic values

### PR DESCRIPTION
**Pull request by pmderodat**

### CODE REVIEW

File: testsuite/tests/ada_api/generic_api/introspection_values.adb [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+   end Put_Exc;
+
+   -------------
+   -- Inspect --
+   -------------
+
+   procedure Inspect (Value : Value_Ref) is
+      Img : constant String := Image (Value);
+   begin
+      --  Print the debig image for this value. The image for analysis units
```
Comment (thvnx) [inline testsuite/tests/ada_api/generic_api/introspection_values.adb:67]: debig -> debug

**Review by thvnx — COMMENTED** [review]
LGTM, as far as I can tell.

File: testsuite/tests/ada_api/generic_api/introspection_values.adb (hunk 1 above)
Comment (pmderodat) [reply testsuite/tests/ada_api/generic_api/introspection_values.adb:67]: Oops, I’ll fix this, thanks!

**Comment by pmderodat** [comment]
> LGTM, as far as I can tell.

Thanks for the review!

**Review by raph-amiard — APPROVED** [review]

**Pull request merged**